### DISCUSSION TITLE
Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: Unable to launch WebDriverAgent because of xcodebuild failure: xcodebuild failed with code 65

### DISCUSSION THREAD

**Discussion by MariappanEsakki** [appium/appium#18975] [General]
https://github.com/appium/appium-xcuitest-driver/blob/master/docs/real-device-config.md. Try to remove the WebDriverAgentRunner application from the device if it is installed and reboot the device.  

I have followed the same still facing the same issue.

**Comment by KazuCocoa** [comment]
Did you confirm from Xcode?
https://discuss.appium.io/ may help to ask questions
